cmd/vsphere-copy-offload-populator/vmkfstools-wrapper/vmkfstools_wrapper.sh

```
@@ -157,10 +157,40 @@ generate_task_id() {
     echo ${hex:0:8}-${hex:8:4}-${hex:12:4}-${hex:16:4}-${hex:20:12}
 }
 
+cleanup_stale_dirs() {
+    log_info "Checking for stale task directories older than 5 days"
+    local stale_dirs
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)